NEWS.txt

```diff
@@ -15,9 +15,9 @@ ImageMagick release, we perform a comprehensive security assessment that
 includes memory error, thread data race detection, and continuous fuzzing
 to detect and prevent security vulnerabilities.
 
-As an analog to linear (RGB) and non-linear (sRGB) color colorspaces, as
```

Comment:
We use `non-linear` every instead so can you revert this? I found [this](https://english.stackexchange.com/a/172298) reference that both are allowed:
> The Oxford Dictionary (which I personally consider the reference for the English language) lists "non-linear" as the correct British spelling and no alternative is given. But if you switch to the US English dictionary, the only entry is "nonlinear".